lib/main.js

```diff
@@ -2,13 +2,16 @@
 
 import { CompositeDisposable } from 'atom';
 import Config from './config';
+const { fixbind } = require('fixbind')
```

Comment:
fix bindings do not support es6?

Reply:
Good catch. It works with both require and import. I can fix it, but IIRC the Atom repo already mixes esm and cjs, so it's just one of many imports that should be corrected.

I'll be happy to convert everything to a single specification. I've already done it for vscode and gotten rid of babel.